### PR TITLE
feat(Flyout): add basic component

### PR DESCRIPTION
**Pull request by NadiaPosch**

### CODE REVIEW

**Comment by triggertoo** [comment]
Badges seem to have the wrong background color in dark mode, but I'm also fine when we fix that in another MR:
badges

File: src/components/Flyout/Flyout.tsx [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+            >
+                {title && (
+                    <>
+                        <div className="tw-flex tw-justify-between tw-flex-wrap tw-gap-3 tw-p-8">
+                            <div {...titleProps} className="tw-inline-flex">
+                                <FieldsetHeader decorator={decorator}>{title}</FieldsetHeader>
+                            </div>
+                            <div className="tw-inline-flex tw-gap-2 tw-flex-wrap">
+                                {badges &&
+                                    badges.map((badge, index) => <Badge key={`flyout-badge-${index}`} {...badge} />)}
```
Comment (triggertoo) [inline src/components/Flyout/Flyout.tsx:65]: rename badge to badgeProps

```suggestion
                                    badges.map((badgeProps, index) => <Badge key={`flyout-badge-${index}`} {...badgeProps} />)}
```

File: src/components/Flyout/Flyout.tsx [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+                )}
+                {Children.map(children, (child, index) => (
+                    <>
+                        {index !== 0 && <Divider color={FLYOUT_DIVIDER_COLOR} height={DividerHeight.Height10} />}
+
+                        <div key={index} className="tw-p-8">
+                            {child}
+                        </div>
+
+                        {index === Children.count(children) - 1 && (
```
Comment (triggertoo) [inline src/components/Flyout/Flyout.tsx:79]: I'd append the Divider after each block/iteration, so you can actually remove line 73

**Comment by dwirz** [comment]
> Badges seem to have the wrong background color in dark mode, but I'm also fine when we fix that in another MR:
> badges

the flyout does not have a dark "theme" atm so the bg of the flyout is probably not optimal

**Review by triggertoo — COMMENTED** [review]
noice

**Pull request merged**